src/styles/mixins/buttons.scss

```diff
@@ -77,7 +77,7 @@
   font-weight: 700;
   background-color: var.$grey-1;
 
-  &:hover {
+  &:hover:not([disabled]) {
     background-color: var.$grey-2;
   }
```

Comment:
I could go either way on this, but I suspect this may be a good rule to have too...

```
&:hover:disabled {
    cursor: default;
  }
```

There is really no reason to suggestion something via cursor style that can't be clicked.

Reply:
I think I'd like to let the browser take control here, and provide whatever cursor it deems appropriate for a `disabled` button. Given that we're selecting on the actual attribute `disabled` (and not, say, an applied class name), we know that this button is in fact disabled...so, let's let the browser do its thing as much as possible.